.github/workflows/tests.yml

```diff
@@ -31,14 +31,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        product:
+          - terramate
+          # - terramate-catalyst
```

Comment:
Is this disabled until catalyst is added to asdf? Should we merge it now as it is or wait until catalyst is added then update and merge it?

Reply:
we can add this now that the repo is public. I will take care